RELEASING.adoc

```diff
@@ -0,0 +1,12 @@
+= Spring Cloud GCP release process
+
+The actual release process is triggered by Pivotal.
+In order to simplify that process, here is a list of prerequisites to be performed before asking Pivotal to run the release scripts.
+
+1. Comment out the `spring-cloud-gcp-samples` module from the root `pom.xml` file, so that the samples aren't released.
+
+2. For any non-GA modules (e.g., Spanner), also comment them out from the `pom.xml` file.
+
+3. Manually change the value of `TRACKING_HEADER_PROJECT_VERSION` in link:spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UsageTrackingHeaderProvider.java[`UsageTrackingHeaderProvider`] to the version being released.
+
+After the release scripts have been run by Pivotal, revert all these changes.
```

Comment:
This line mentions scripts being run by Pivotal, implying that that process is outside of the process described herein. As a result, I don't think you can remove the first line in this doc if you're going to keep this one, because it would be without context.

Reply:
Wherever you're saying "Pivotal" right now, you can just say "project admins".

Reply:
@meltsufin that could be less informative. I can imagine myself as new to the team and thinking we are the project admins since we are in charge of code reviews and writing the code. Saying "Pivotal" is totally clear and probably will stay accurate for the foreseeable future .

Reply:
Well, then this document should be internal for your team.
One more time: it doesn't make sense to say in the Pivtoal doc that we need to ask Pivotal.

This project is Pivotal proprietary and it is really clear from context of the project that who is responsible for release process.

Well, we really can say `project admins` and then do not consider your team as in the admin role.
You are indeed not admins: you don't perform release 🤷‍♂️